### PR TITLE
Fix broken McAfee links

### PR DESCRIPTION
**Pull request by IPv4v6**

### CODE REVIEW

**Comment by kingthorin** [comment]
Are you sure these aren’t actually still available from McAfee via different URLs?

**Comment by IPv4v6** [comment]
It seems, McAfee removed the hackable applications and I wasn't able to find these on McAfee's website.

File: src/data/offline.json [hunk 1]
```diff
@@ -433,7 +433,7 @@
 		"references": [
 			{
 				"name": "download",
-				"url": "http://www.mcafee.com/apps/free-tools/termsofuse.aspx?url=/us/downloads/free-tools/hacme-bank.aspx"
+				"url": "http://web.archive.org/web/20130204025614if_/http://downloadcenter.mcafee.com/products/tools/foundstone/hacmebank2_install.zip"
```
Comment (kingthorin) [inline src/data/offline.json:436]:
```suggestion
				"url": "http://downloadcenter.mcafee.com/products/tools/foundstone/hacmebank2_install.zip"

```

File: src/data/offline.json [hunk 2]
```diff
@@ -460,7 +460,7 @@
 		"references": [
 			{
 				"name": "download",
-				"url": "http://www.mcafee.com/apps/free-tools/termsofuse.aspx?url=/us/downloads/free-tools/hacmebooks.aspx"
+				"url": "http://web.archive.org/web/20130924075344if_/http://b2b-download.mcafee.com/products/tools/foundstone/hacmebooks2_installer.zip"
```
Comment (kingthorin) [inline src/data/offline.json:463]:
```suggestion
				"url": "http://b2b-download.mcafee.com/products/tools/foundstone/hacmebooks2_installer.zip"

```

File: src/data/offline.json [hunk 3]
```diff
@@ -476,7 +476,7 @@
 		"references": [
 			{
 				"name": "download",
-				"url": "http://www.mcafee.com/apps/free-tools/termsofuse.aspx?url=/us/downloads/free-tools/hacme-casino.aspx"
+				"url": "http://web.archive.org/web/20130204042019if_/http://downloadcenter.mcafee.com/products/tools/foundstone/hacmecasino_installer.zip"
```
Comment (kingthorin) [inline src/data/offline.json:479]:
```suggestion
				"url": "http://downloadcenter.mcafee.com/products/tools/foundstone/hacmecasino_installer.zip"

```

File: src/data/offline.json [hunk 4]
```diff
@@ -492,7 +492,7 @@
 		"references": [
 			{
 				"name": "download",
-				"url": "http://www.mcafee.com/apps/free-tools/termsofuse.aspx?url=/us/downloads/free-tools/hacmeshipping.aspx"
+				"url": "http://web.archive.org/web/20130204042010if_/http://downloadcenter.mcafee.com/products/tools/foundstone/hacmeshipping.zip"
```
Comment (kingthorin) [inline src/data/offline.json:495]:
```suggestion
				"url": "http://downloadcenter.mcafee.com/products/tools/foundstone/hacmeshipping.zip"

```

File: src/data/offline.json [hunk 5]
```diff
@@ -508,7 +508,7 @@
 		"references": [
 			{
 				"name": "download",
-				"url": "http://www.mcafee.com/apps/free-tools/termsofuse.aspx?url=/us/downloads/free-tools/hacmetravel.aspx"
+				"url": "http://web.archive.org/web/20130204042023if_/http://downloadcenter.mcafee.com/products/tools/foundstone/hacmetravel_install.zip"
```
Comment (kingthorin) [inline src/data/offline.json:511]:
```suggestion
				"url": "http://downloadcenter.mcafee.com/products/tools/foundstone/hacmetravel_install.zip"

```

**Review by kingthorin — CHANGES_REQUESTED** [review]
Use of the Wayback Machine seems un-necesary, they're all still available from the referenced links.

**Review by kingthorin — APPROVED** [review]

**Pull request merged**